extensions/src/ACRE2TS/TsCallbacks_static.cpp

```diff
@@ -34,27 +34,33 @@ void ts3plugin_infoData(uint64 serverConnectionHandlerID, uint64 id, enum Plugin
 
     BOOL noAcre = FALSE;
     char *metaData;
+    std::string result;
+    std::string sharedMsg;
 
     switch(type) {
         case PLUGIN_SERVER:
             break;
         case PLUGIN_CHANNEL:
             break;
         case PLUGIN_CLIENT:
-            if (ts3Functions.getClientVariableAsString(serverConnectionHandlerID, (anyID)id, CLIENT_META_DATA, &metaData) == ERROR_ok) {
-                if (!metaData) { 
-                    noAcre = TRUE;
-                }
-                int metaDataLength = strlen(metaData);
-                if (metaDataLength < 3 || metaDataLength > (INFODATA_BUFSIZE - 2)) {
+            // Some code used from https://github.com/michail-nikolaev/task-force-arma-3-radio under APL-SA
```

Comment:
APL SA is not compatible with GPL. Either this is refactored or this PR cannot be merged.

Reply:
It's refactored quite a bit, but it's still recognizable as from TFAR. Didn't realize ACRE was GPL and not APL-SA. I can rewrite this at some point. Optionally would we be able to get permission from the author of this code to use it under GPL for ACRE?

Reply:
I am sure there are other examples on how to add and use tags

Reply:
It's not a matter of difficulty, this isn't a hard thing to fix in the slightest. I just thought if I've already used the TFAR code for inspiration and got it fixed it may just be quicker to ask if we can use it rather than rewriting it. If that is not possible I can refactor it.

Reply:
It is better that no code from TFAR enters acre2. We do not even have to take the same approach and if there is a better solution to this then we take it. Compatibility between plugins is not a top priority since loading tfar and and crew is not really an option :D

Reply:
I'm not doing this to have both TFAT and ACRE used at the same time. ACRE will erase data from any TeamSpeak plugin, not just TFAR. A benefit of this is that both TeamSpeak plugins can be left enabled. As someone that does joint ops with groups using TFAR but uses ACRE in my group, this will let me leave both plugins enabled rather than disabling and enabling all the time.